mungegithub/submit-queue/www/index.html

```diff
@@ -248,6 +248,21 @@ <h2 class="md-toolbar-tools">Health chart for the past week</h2>
                     <span><img ng-src="{{cntl.metadata.ChartUrl}}" style="max-width: 100%"/></span>
                   </md-content>
                 </md-tab>
+
+                <md-tab label="Bot Commands">
+                    <h1>Communicating with the Bot</h1>
+
+                Github contributors and reviewers can communicate with the mungebot by commenting on a PR with the
+                  following commands entered alone in the text field. All commands require the following syntax:
+                  <br><br> /Command [Optional Args]. <br><br> Note the forward slashing preceding the command name.
+                    <h2>List of Commands</h2>
+
+                    <ol>
+                      <li>lgtm: applies the lgtm label </li>
+                      <li> lgtm cancel: removes a previously applied lgtm label </li>
+                    </ol>
+
+                </md-tab>
```

Comment:
I know that I did `queue order` and `merge requirements` from inside submit-queue.go because it was the single place the information made sense.

If we expect all (or most) of the commands to be implement/used in submit-queue.go it might make sense to do that. It would likely be best to have all of the (running) mungers register commands which would be outputted.

But I'll merge this patch if others are happy and think we can iterate rather than do it up front...